plugins/telemetry/telemetry_impl.cpp

```diff
@@ -1,37 +1,39 @@
 #include "telemetry_impl.h"
-#include "system.h"
-#include "math_conversions.h"
-#include "global_include.h"
-#include "px4_custom_mode.h"
+
 #include <cmath>
 #include <functional>
 
+#include "global_include.h"
+#include "math_conversions.h"
+#include "px4_custom_mode.h"
+#include "system.h"
+
 namespace dronecore {
 
 TelemetryImpl::TelemetryImpl(System &system) :
     PluginImplBase(system),
     _position_mutex(),
-    _position(Telemetry::Position {double(NAN), double(NAN), NAN, NAN}),
+    _position(Position {double(NAN), double(NAN), NAN, NAN}),
     _home_position_mutex(),
-    _home_position(Telemetry::Position {double(NAN), double(NAN), NAN, NAN}),
+    _home_position(Position {double(NAN), double(NAN), NAN, NAN}),
     _in_air(false),
     _armed(false),
     _attitude_quaternion_mutex(),
-    _attitude_quaternion(Telemetry::Quaternion {NAN, NAN, NAN, NAN}),
+    _attitude_quaternion(Quaternion {NAN, NAN, NAN, NAN}),
```

Comment:
While you're at it I suggest to do all these default values in the header file instead of the constructor initializer list.